src/components/interaction-designer/Block.vue

```diff
@@ -622,6 +623,23 @@ export default {
   transition: opacity 200ms ease-in-out,
   background-color 200ms ease-in-out;
 
+  .block-toolbar {
+    transition: opacity 200ms ease-in-out;
+    background: white;
+    opacity: 0; // default state of hidden
+
+    margin-top: -39.75px;
+    margin-right: -8px;
+    margin-left: -8px;
```

Comment:
I used negative margins to take what was inside and pull the whole block contents outside of the .block container.